VAMobile/documentation/docs/Engineering/FrontEnd/DevelopmentBestPractices.md

```diff
@@ -0,0 +1,130 @@
+
+## Tickets
+
+We use tickets to track all our engineering work. They precisely specify the work to be performed, allow stakeholders to track progress, and guide QA in assessing the resulting work.
+
+### Creating tickets
+
+Every ticket must contain at least the following information:
+
+- A detailed description of the work
+- Clearly specified Acceptance Criteria
+- Screenshots or links to mockups where applicable
+- All relevant labels ("front-end" for FE work, "Health" for Health and Benefits work, "Global" for global work, etc)
+- An epic relating to the ticket's larger context
+
+If possible, choose the relevant template for your ticket. For example, if you're creating a bug ticket, choose the Bug Report Template and include severity in the title, labels, and body.
+
+### Working on tickets
+
+All engineering work requires a ticket.
+
+Every ticket must satisfy the requirements in the "Creating tickets" section above. If information is missing, add it or contact the relevant stakeholders.
+
+Before an engineer can begin work on a ticket, it must contain an estimate. Bugs get an initial estimate of 1 to cover a short investigation. Other tickets are left to the engineers' discretion.
+
+Once everything is in place, complete the following tasks:
+
+- Assign yourself
+- Assign the appropriate QA engineer if not already assigned
+- Move the ticket to In Progress
+- Create a branch as described below in the Branches section
+
+Then you can begin work. Keep stakeholders up to date as you work by commenting on the ticket with any important details or questions. This will also preserve information so everyone can reference it in the future.
+
+On completion of your work, double check the following:
+
+- All items in Acceptance Criteria are implemented
+- Confirm screen screen reader functionality on both iOS and Android for accessibility
+- Unit and E2E tests are updated or extended to cover your changes
+- Relevant documentation is updated to reflect the changes in your PR
+
+Then create a Pull Request. See the Pull Requests section below for more information.
+
+## Branches
+
+Feature branches are temporary branches where we develop and test new features. We implement every code change in a feature branch before merging it into the default branch.
+
+:::info
+For features that must be rolled out gradually, or where precise feature activation timing is required, use a [remote config flag](https://firebase.google.com/docs/remote-config/).
+:::
+
+Our default branch is `develop`. Create your feature branch from there, named as follows:
+
+```
+type/ticketnumber-description-of-work
+```
+
+The ticket number lets us track back to the relevant ticket. The type and description show the branch's purpose at a glance. Here are some example branch names:
+
+- `feature/1234-description-of-feature-being-done`
+- `chore/1234-description-of-chore-being-done`
+- `bugfix/1234-description-of-bug-being-fixed`
```

Comment:
```suggestion
- `bug/1234-description-of-bug-being-fixed`
```
Feel like most (all?) of us just use "bug" and the "fix" feels redundant since hopefully we aren't making branches to add bugs. 😆

Reply:
Sure, updated